packages/components/src/components/modal/modal-styles.ts

```diff
@@ -219,6 +241,10 @@ export const getComponentCss = (
             borderColor: darkThemeContrastHighColor,
           },
         }),
+        ...prefersColorSchemeDarkMediaQuery(theme, {
+          background: backgroundColorDark,
+          border: `2px solid ${backgroundColorDark}`,
```

Comment:
Could just overwrite `border-color` instead of entire `border` shorthand.

Reply:
it's stronger and should be the correct one